### ISSUE TITLE
Qt 5.5 branch requires Qt 5.6?

### ISSUE THREAD

**Issue by Vistaus** [QupZilla/qupzilla#1791]
So I was trying to build the Qt 5.5 branch (I checked it out using git checkout qt55) but after running qmake it says that it needs Qt 5.6 How is it possible that the Qt 5.5 branch needs 5.6?

**Comment by cranes-bill** [comment]
A quite easy, when you checked out `MASTER` instead.....

**Comment by Vistaus** [comment]
But I switched to the 5.5 branch. When I try to checkout the 5.5 branch again, Git says that I'm already on the 5.5 branch. There's no trace of Master anywhere...

**Comment by nowrep** [comment]
It's `qt5.5`, you created new branch from HEAD instead.

**Comment by Vistaus** [comment]
When I try t checkout qt5.5 it says that the branch doesn't exist.

**Comment by nowrep** [comment]
https://github.com/QupZilla/qupzilla/tree/qt5.5

**Comment by Vistaus** [comment]
Strange, very strange. I deleted the cloned QupZilla folder, cloned the QupZilla Master branch again (clean) and it still complained that the qt5.5 branch didn't exist. Removed again, checked out Master again and then it finally let me switch to the qt5.5 branch. Seems like Git has a bug or two... it can't be my install, 'cause I installed Antergos over Mint yesterday, so the OS install is 100% clean...

Thanks for helping anyway! :)
